package.json

```diff
@@ -64,7 +64,9 @@
     "enzyme/cheerio/cheerio-select-tmp/css-what": "^6.1.0",
     "html-webpack-plugin/**/ansi-regex": "^5.0.1",
     "jest-cli/**/ansi-regex": "^5.0.1",
+    "jest-cli/**/tough-cookie": "^4.1.3",
     "jest/**/node-notifier": "^10.0.1",
+    "jest/**/tough-cookie": "^4.1.3",
```

Comment:
@BSFishy Is it possible to solve this with a bump to `jest` and `jest-cli`, or have they not yet released versions with the fix?

Reply:
+1

Reply:
These deps are a couple major versions behind. Last time I tried updating them there were some pretty big issues. I think updating them will be a bigger effort than a quick version bump